### DISCUSSION TITLE
Old beta maps

### DISCUSSION THREAD

**Discussion by Hutch69** [nzp-team/nzportable#373] [Map Releases]
I downloaded these about 100 years ago on the nzp forums. Some of them are quite good, other not so much. Google drive file is the whole release. I tried to upload only the maps however it exceeded the file limit here.

[Google Drive files here](https://drive.google.com/file/d/0B_yKc7T0OKa6NTBfQ1Q4cGRObU0/view?usp=sharing&resourcekey=0-Ky96zRLbMekUlWLSjbYdMQ)

<img width="593" alt="image" src="https://github.com/nzp-team/nzportable/assets/144613030/7a843747-d0d8-479b-bf36-7c64cc2fad09">

**Comment by DerpedCrusader** [comment]
ZOMBIE ISLAND LETS GOOOOO

that map was legit lost to time

And it was my favorite as a kid

**Reply by actraiser30** [reply]
Hi any fru mansion map here?

**Reply by DerpedCrusader** [reply]
https://docs.nzp.gay/landing/

It's not going to make itself homie :3 the best way to get good at mapping is to start doing it, we all start somewhere, you just gotta give it a shot

**Reply by actraiser30** [reply]
No way out of my skills

**Reply by DerpedCrusader** [reply]
> No way out of my skills


Only one way to get the skills, im sorry homie you're the only person interested in this existing, so you've gotta step up to the plate

**Comment by oshogaps-glitch** [comment]
waypoints lot of these are pointless with no waypoint

**Reply by endis-offical** [reply]
You can make your own waypoints

**Reply by oshogaps-glitch** [reply]
yea you right but only a few maps are really good most are bugged i went thru all and tested only playing those maps ig


(some odd reason) when i open waypoint mode and = these already had waypoints for them just saved it (these work with little to no glitches)

myhouse
ZombieIsland
Bakaara
broken
condemned
dung3on
fghf
hell
het
likederriese
mix
mt 



(these maps dont glitch either just dont have waypoints so zombies are dumb if anyone makes them and wants to drop it please and ty)

TheLastStand
Zoschrin
BoxHouse
yard2
zombiehouse
Ansteckung
1room
Zimmer
GRIDYARD
killsonik
Containment
Perk-A-Station
BestMapEver
Temple
Compact
Reticent
tinysurvival
2nd
uhhm
roof
small_map
kein
purgatory
the_mansion
bigj
house
zombiesmaybe
lobby
loop
minecraft
gCroom
Olympus
compact2


(these beta maps str8 up dont work and are just broken no need playing them zombies dont go up stairs or down and gets blocked by invisiable barriers)


4_2
haus
undrgrnd_base
smallfactory
kirche_der_untoten
tag_der_toten
yard
Old
Gijutsu_naitomea
gunstore
TodeShaus
apartment
MAP2
nighthouse
ZimmerDerToten
SURVIVE
cristmas_factory
xmas_special
death_factory

**Reply by oshogaps-glitch** [reply]
ps i can write my own waypoints but everytime i do shit always fails lol like everytime

**Reply by Hutch69** [reply]
A lot of these maps were made by young people just getting started into the world of mapping back when this was a PSP homebrew. Skill levels vary greatly, and chances are not all of them were completed when I downloaded them from the nzp forums.

**Reply by M0THRAK** [reply]
Thanks for making this list, excited to check them out!

**Comment by actraiser30** [comment]
I don't even have a computer anymore ahahaha no way
